cli/src/main.rs

```diff
@@ -387,6 +390,16 @@ fn main() -> Result<(), io::Error> {
         })
         .init();
 
+    if max_level() >= LevelFilter::Debug {
```

Comment:
Why does it use Debug as the filter here but shows INFO in the PR comment with the example?

Reply:
ah ok